### ISSUE TITLE
Add digital calipers

### ISSUE THREAD

**Issue by ctrlcctrlv** [mfeq/Qglif#18]
Requires #13.

Cf. http://www.corvelsoftware.co.uk/software/supertool/

**Comment by ctrlcctrlv** [comment]
This is in 1.0 since it's really easy to do with a measure tool and I want to have some features in 1.0 impossible in @FontForge so people get it. Literally just in a loop interpolate a line and use Skia to measure the intersected portion.

**Comment by davelab6** [comment]
I agree this is high priority, I believe beginners really benefit massively from this feature, as they can't yet see letterforms closely enough to gauge what the calipers tool makes more visual.

**Comment by ctrlcctrlv** [comment]
I hired Matt to close this issue
